TestCases/optimization_euler/equivalentarea_naca64206/NACA64206.cfg

```diff
@@ -0,0 +1,434 @@
+MARKER_PLOTTING= ( Wing )
+%
+% Marker of the surface where the functional (Cd, Cl, etc.) will be evaluated
+MARKER_MONITORING= ( Wing )
+%
+% Near-Field boundary marker(s) (NONE = no marker)
+MARKER_NEARFIELD= ( Nearfield )
+
+% ------------- COMMON PARAMETERS TO DEFINE THE NUMERICAL METHOD --------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= GREEN_GAUSS
+
+%
+% Objective function in gradient evaluation  (DRAG, LIFT, SIDEFORCE, MOMENT_X,
+%                                             MOMENT_Y, MOMENT_Z, EFFICIENCY,
+%                                             EQUIVALENT_AREA, NEARFIELD_PRESSURE,
+%                                             FORCE_X, FORCE_Y, FORCE_Z, THRUST,
+%                                             TORQUE, FREE_SURFACE, TOTAL_HEATFLUX,
+%                                             MAXIMUM_HEATFLUX, INVERSE_DESIGN_PRESSURE,
+%                                             INVERSE_DESIGN_HEATFLUX)
+OBJECTIVE_FUNCTION= EQUIVALENT_AREA
+%
+% Courant-Friedrichs-Lewy condition of the finest grid
+CFL_NUMBER=3
+%
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= NO
+%
+% Parameters of the adaptive CFL number (factor down, factor up, CFL min value,
+%                                        CFL max value )
+CFL_ADAPT_PARAM= ( 0.1, 2.0, 3, 100.0, 0.001 )
+%
+% Runge-Kutta alpha coefficients
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+%
+% Number of total iterations
+INNER_ITER= 4000
+%
+% Linear solver for the implicit formulation (BCGSTAB, FGMRES)
+LINEAR_SOLVER= FGMRES
+
+%
+% Preconditioner of the Krylov linear solver (ILU, LU_SGS, LINELET, JACOBI)
+%
+LINEAR_SOLVER_PREC=LU_SGS
+%
+% Min error of the linear solver for the implicit formulation
+LINEAR_SOLVER_ERROR= 1E-10
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 50
+
+% ----------------------- SLOPE LIMITER DEFINITION ----------------------------%
+%
+MUSCL_FLOW=YES
+SLOPE_LIMITER_FLOW=VAN_ALBADA_EDGE
+MUSCL_ADJFLOW=YES
+SLOPE_LIMITER_ADJFLOW=VAN_ALBADA_EDGE
+% Coefficient for the limiter
+VENKAT_LIMITER_COEFF= 0.03
+%
+% Coefficient for the sharp edges limiter
+ADJ_SHARP_LIMITER_COEFF= 3.0
+%
+% Reference coefficient (sensitivity) for detecting sharp edges.
+REF_SHARP_EDGES= 3.0
+%
+% Remove sharp edges from the sensitivity evaluation (NO, YES)
+SENS_REMOVE_SHARP= YES
+
+% -------------------------- MULTIGRID PARAMETERS -----------------------------%
+%
+% Multi-Grid Levels (0 = no multi-grid)
+MGLEVEL= 0
+%
+% Multi-grid cycle (V_CYCLE, W_CYCLE, FULLMG_CYCLE)
+MGCYCLE= W_CYCLE
+%
+% Multi-Grid PreSmoothing Level
+MG_PRE_SMOOTH= ( 1, 2, 3, 3 )
+%
+% Multi-Grid PostSmoothing Level
+MG_POST_SMOOTH= ( 0, 0, 0, 0 )
+%
+% Jacobi implicit smoothing of the correction
+MG_CORRECTION_SMOOTH= ( 0, 0, 0, 0 )
+%
+% Damping factor for the residual restriction
+MG_DAMP_RESTRICTION= 0.9
+%
+% Damping factor for the correction prolongation
+MG_DAMP_PROLONGATION= 0.9
+
+
```

Comment:
```suggestion
MGLEVEL= 0
```
And so on

Reply:
See below.
https://github.com/su2code/SU2/pull/1329#discussion_r673786199